musicHelper.md

```diff
@@ -0,0 +1,128 @@
+#Piano Music Helper
+
+###_Introduction_
+This bot helps you find a piano note of the famous word composers 
+
+at pdf or a doc documents for printing it for your device
+
+
+##User story for people who play with piano
+As a telegram bot user
+
+I want to find a note with good type of docs
+
+So that I can  print it or download on my devise
+
+And play at my piano
+
+###_start of the screen scenario_
```

Comment:
Please do not use underscores and start your name your scenarios with `Scenario 1: ...`